tools/packaging/boms/windows.json

```diff
@@ -1,14 +1,4 @@
 [
-  {
-    "Pattern": "_manifest\\spdx_2.2\\bsi.json",
-    "FileType": "NonProduct",
-    "Architecture": null
-  },
-  {
-    "Pattern": "_manifest\\spdx_2.2\\manifest.cat",
-    "FileType": "NonProduct",
-    "Architecture": null
-  },
```

Comment:
The logic that was putting these files in must be broken too?

Reply:
Added those 2 entries back.